service/create/pki.go

```diff
@@ -0,0 +1,98 @@
+package create
+
+import (
+	"fmt"
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/certctl/service/pki"
+	"github.com/giantswarm/certctl/service/token"
+)
+
+// Creates a PKI backend if one does not exist for the cluster.
```

Comment:
Please see https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences. The comment should start with `setupPKIBackend ...`.